_posts/pl/2023-10-22-kouncil-1.6.md

```diff
@@ -0,0 +1,62 @@
+---
+layout:    post
+title:     "Kouncil - uprawnienia"
+date:      2023-10-22 7:00:00 +0100
+published: true
+didyouknow: false
+lang: pl
+lang-ref:  kouncil-1.6
+author:    pbelke
+image:     /assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6.png
+description: "Wersja 1.6 Kouncil wprowadza konfigurację uprawnień, umożliwiając precyzyjne kontrolowanie dostępu użytkowników do różnych funkcji systemu."
+tags:
+- kouncil
+- programming
+- kafka
+- authorization
+---
+
+Najważniejszą zmianą w wersji 1.6 Kouncil jest konfiguracja uprawnień, która umożliwia precyzyjne kontrolowanie dostępu użytkowników do funkcji systemu. Ponadto, w ramach ciągłego doskonalenia interfejsu aplikacji, wprowadziliśmy kilka drobniejszych zmian ułatwiających pracę w systemie.
+
+## Konfiguracja uprawnień
```

Comment:
To jest de factor autoryzacja (eng. authorization) ale w tekście nie pada ani razu, ani w PL ani w ENG poza tagami i linkami. Nie upieram się, bo opis jest prawidłowy, ale jak by się udało wpleść gdzie te terminy to by było ok. Może w tytule sekcji?